t/lang.t

```diff
@@ -79,4 +79,22 @@ sub test_logo_exists {
 test_logo_exists('logo');
 test_logo_exists('logo2x');
 
+# Test that {variables} are kept in translations
+
+foreach my $stringid (sort keys %Lang) {
+	while ($Lang{$stringid}{en} =~ /{([^}]+)\}/g) {
```

Comment:
Perl has some surprising syntax ! (the while with a match that will automatically iterate through matches…)

Reply:
Even if not mandatory, I think adding a backslash to the first curlybrace is more explicit (and still works)

```suggestion
	while ($Lang{$stringid}{en} =~ /\{([^}]+)\}/g) {
```